### PR TITLE
AEA-723 Readme Files for Protocol Packages

### PR DESCRIPTION
**Pull request by 5A11**
This PR adds a readme.md file in every protocol package. 

It lays the ground for agreeing on a template for such readme files. Thus, it is currently a work in progress, only adding readme for the Fipa protocol for now.

## Types of changes

What types of changes does your code introduce to agents-aea?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply._

- [x] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [x] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that code coverage does not decrease.
- [ ] I have checked that the documentation about the `aea cli` tool works
- [x] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: packages/fetchai/protocols/fipa/README.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+**Author**: fetchai
+
+**Version**: 0.4.0
+
+**Short Description**: A protocol for FIPA ACL.
+
+**License**: Apache-2.0
+
+**AEA Framework Versions Supported**: 0.5.0 to 0.6.0
```
Comment (DavidMinarsch) [inline packages/fetchai/protocols/fipa/README.md:13]: This is not needed; we will display it automatically on the website.

File: packages/fetchai/protocols/fipa/README.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+termination: [decline, match_accept, match_accept_w_inform, inform]
+roles: {seller, buyer}
+end_states: [successful, declined_cfp, declined_propose, declined_accept]
+...
+```
+
+## Links
+
+* <a href="http://www.fipa.org" target=_blank>FIPA Foundation</a>
+* <a href="#" target=_blank>Report</a>
```
Comment (DavidMinarsch) [inline packages/fetchai/protocols/fipa/README.md:76]: where is the second link to?

Comment (5A11) [reply packages/fetchai/protocols/fipa/README.md:76]: Nothing. 

This is just to demonstrate that one can put links to other resources, e.g. papers, repos, related to the protocol here. For FIPA I remove it as we don't have any further resources yet.

Comment (marcofavorito) [reply packages/fetchai/protocols/fipa/README.md:74]: I suggest using 
```
[FIPA Foundation](http://www.fipa.org)
```
Instead of HTML.

Comment (5A11) [reply packages/fetchai/protocols/fipa/README.md:74]: What's the advantage? 

Doesn't the HTML way give more control, e.g. to open the link in a new/existing tab?

**Pull request merged**